BonovoBluetooth/src/com/bonovo/bluetooth/BonovoBluetoothHandfree.java

```diff
@@ -359,49 +387,59 @@ protected void onResume() {
 			}
 		}
 		mDialpadPressCount = 0;
+		

```

Comment:
Ok, so I got a chance to test in the car as I drove to work this morning, and it crashes as soon as the phone connects.. 
I'll move the computer out to the garage tonight and sort it out. I'm jealous of you guys with more than one headunit, testing would be a lot easier inside :-)